packages/legend-studio/src/stores/EditorGraphState.ts

```diff
@@ -1016,6 +1021,53 @@ export class EditorGraphState {
     return dependencyEntitiesMap;
   }
 
+  *buildProjectDependencyCoordinates(
+    projectDependencies: ProjectDependency[],
```

Comment:
By doing this, you're disregarding the order of dependencies being specified in the project structure. I know order doesn't really play a role right now, but we probably should still respect such order.